src/module-1/card/style.css

```diff
@@ -0,0 +1,100 @@
+html {
+    background-color: whitesmoke;
+}
+
+
+.card{
+    width: 400px;
+    font-family: 'Montserrat', sans-serif;
+    background-color: white;
+    border-radius: 8px;
+}
+
+.card__rating{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.card-rating__stars{
+    height: 30px;
+    background-color: #6F64F8;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    color: white;
+    width: 65px;
+}
+
+.card-rating__price {
+    font-size: 1.1em;
+}
+
+.card__bottom{
+    display: flex;
+    flex-direction: row;
+}
+
+.card__description{
+    padding-top: 16px;
+}
+
+.card-description__title{
+    font-size: 1em;
+    line-height: 19px;
+    margin-bottom: 3px;
```

Comment:
please consider using even values for dimensions where possible (unless we use odds for centering or some very specific cases), that's a good practice